backend/background/sources/collective_bargaining.py

```
@@ -13,196 +13,267 @@
+
+            if is_campus:
+                # This is a local campus accordion, skip it in systemwide processing
+                logger.info(
+                    f"Skipping campus accordion in systemwide: {union_name}")
+                continue
+
+            # Click to expand the accordion if it's not already expanded
+            try:
+                await button.click()
+                await page.wait_for_timeout(500)  # Wait for expansion
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)